src/components/uploadPicture/uploadImage.tsx

```diff
@@ -39,31 +34,23 @@ const UploadImageComponent: React.FC<UploadImageComponentProps> = ({
   const handleFile = (event: React.ChangeEvent<HTMLInputElement>) => {
     const file = event.target.files?.[0];
     if (file) {
-      const reader = new FileReader();
-      reader.onload = () => {
-        const fileContent = reader.result;
-        if (fileContent instanceof ArrayBuffer) {
-          const fileBlob = new Blob([fileContent]);
-          setCapturedImage(fileBlob);
-        }
-      };
-      reader.readAsArrayBuffer(file);
+      setSelectedFile(file);
       setCurrentStep(UploadStep.UploadPicture);
     }
   };
 
   const onCancelTakePicture = () => {
     setCurrentStep(UploadStep.Start);
-    setCapturedImage(null);
+    setSelectedFile(null);
   };
 
   const onClickUploadPicture = () => {
     setCurrentStep(UploadStep.Start);
-    setCapturedImage(null);
+    setSelectedFile(null);
     setUploadSuccess(true);
     setUploadSuccess(false);
```

Comment:
Not your code, but do you know why this is setting to true then false immediately?

Reply:
not sure, I'll clean up this file a little bit

Reply:
0825676